janusgraph-backend-testutils/src/main/java/org/janusgraph/core/attribute/TextArgument.java

```diff
@@ -0,0 +1,474 @@
+// Copyright 2017 JanusGraph Authors
```

Comment:
2011

Reply:
Just saw this, updated the Copyright on the new file to 2021.